package.yaml

```diff
@@ -36,10 +38,11 @@ library:
     base: '>= 4.7 && < 5'
     bytestring: '>= 0.10.8 && < 0.11'
     directory: '>= 1.3 && < 1.4'
+    extra:
```

Comment:
i'd prefer not to drag in the huge `extra` library. would you be ok with implementing `concatMapM` manually?

Reply:
done and re-ran tests locally